### PR TITLE
[15.0][FIX] account_credit_control

### PR DESCRIPTION
**Pull request by dsolanki-initos**
message_format() got an unexpected keyword argument 'format_reply'

### CODE REVIEW

File: account_credit_control/models/mail_message.py [hunk 1]
```diff
@@ -7,8 +7,8 @@
 class MailMessage(models.Model):
     _inherit = "mail.message"
 
-    def message_format(self):
-        result = super().message_format()
+    def message_format(self, format_reply=True):
+        result = super().message_format(format_reply)
```
Comment (pedrobaeza) [inline account_credit_control/models/mail_message.py]: Respect keyword arguments

```suggestion
        result = super().message_format(format_reply=format_reply)
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Change also message commit to comply with guidelines:

`[FIX] account_credit_control: Params of message_format`

File: account_credit_control/models/mail_message.py (hunk 1 above)
Comment (dsolanki-initos) [reply account_credit_control/models/mail_message.py]: @pedrobaeza , I've fixed the commit message and keyword arguments as you mentioned above. Please review it.

**Review by pedrobaeza — APPROVED** [review]
The fix is clear:

https://github.com/odoo/odoo/blob/c55badb8d70bdccfeb82bffb330d3f0b1b162ed1/addons/mail/models/mail_message.py#L911

so fast-tracking it:

/ocabot merge patch

**Pull request merged**